modules/3d-tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -26,15 +27,44 @@ const scratchTransform = new Matrix4();
 export function createBoundingVolume(boundingVolumeHeader, transform, result) {
   assert(boundingVolumeHeader, '3D Tile: boundingVolume must be defined');
   if (boundingVolumeHeader.box) {
-    return null;
+    // The first three elements define the x, y, and z values for the center of the box.
+    const [x, y, z] = boundingVolumeHeader.box;
+    let center = new Vector3(x, y, z);
+    center = new Matrix4(transform).transformVector(center);
+    center = Ellipsoid.WGS84.cartesianToCartographic(center, center);
+
+    Object.assign(boundingVolumeHeader, {center});
```

Comment:
I am thinking we should avoid modifying the actual loaded data. We could store this on the tileset class root itself.